tests/unit/test_models.py

```diff
@@ -67,35 +66,37 @@ def test_validate_grammar_simple():
             """
             control: a string
             grammar_str:
-              on amd64: another string
-              else: something different
+              - on amd64: another string
```

Comment:
sorry I missed this the first time!

Reply:
Also the "else fail" clause becomes ugly if it's a dictionary ("else fail:").